test/functional.js

```diff
@@ -617,6 +617,40 @@ module.exports = {
             });
         });
 
+        it('TypeScript is compiled and type checking is done in separate process!', (done) => {
+            this.timeout(8000);
+            setTimeout(done, 7000);
+
+            const config = createWebpackConfig('www/build', 'dev');
+            config.setPublicPath('/build');
+            config.addEntry('main', ['./js/render.ts', './js/index.ts']);
+            config.enableTypeScriptLoader();
+            config.enableForkedTypeScriptTypesChecking({
+                tsconfig: './js/tsconfig.json',
+                silent: true // remove to get output on terminal
+            });
+
+            testSetup.runWebpack(config, (webpackAssert) => {
+                // check that ts-loader transformed the ts file
+                webpackAssert.assertOutputFileContains(
+                    'main.js',
+                    'document.getElementById(\'app\').innerHTML = "<h1>Welcome to Your TypeScript App</h1>";'
+                );
+
+                expect(config.outputPath).to.be.a.directory().with.deep.files([
+                    'main.js',
+                    'manifest.json'
+                ]);
+
+                done();
+
+                // TODO find a way to confirm forked is used. Maybe terminal output?
+                // Output:
+                // Starting type checking service...
+                // Using 1 worker with 2048MB memory limit
```

Comment:
hmm. Well, we have other tests that confirm that the config is setup correctly. Is it enough to know just that this test didn't blow up? Is there any reasonable way it could blow up? I'm probably ok without adding anything additional.